pyramid_chameleon/precompile.py

```diff
@@ -33,20 +41,19 @@ def walk_dir(
             if ext not in extensions:
                 continue
             fullpath = os.path.join(dirpath, filename)
-            try:
-                compile_one(fullpath, template_factory=template_factory)
-            except:
-                if fail_fast:
-                    raise
-                yield dict(
-                    path=fullpath,
-                    success=False)
-                logging.warn('Failed to compile: %s' % fullpath)
-                continue
-            logging.debug('Compiled: %s' % fullpath)
-            yield dict(
-                path=fullpath,
-                success=True)
+            ret.append(fullpath)
+    return ret
+
+def walk_dir(
+        directory,
+        extensions=frozenset(['.pt']),
+        template_factory=PyramidPageTemplateFile,
+        fail_fast=False,
```

Comment:
Where is `fail_fast` used in this function? If it is no longer used/useful then maybe it should be removed.

Reply:
`fail_fast` is passed as an argument to use in `_compile_one` function.